cfdilib/templates/cfdi13moves.xml

```diff
@@ -17,7 +17,7 @@
     <PLZ:Poliza Concepto="{{ move.name }}" Fecha="{{ move.date }}" NumUnIdenPol="{{ move.id }}">
     {% for line in move.line_ids %}
         <PLZ:Transaccion Concepto="{{ line.name }}" DesCta="{{ line.account_id.name }}"
-                         NumCta="{{ line.account_id.code }}" Debe="{{ line.debit }}" Haber="{{ line.credit }}">
+                         NumCta="{{ line.account_id.code }}" Debe="{{ inv.formatLang(move.env, line.debit, digits=2) }}" Haber="{{ inv.formatLang(move.env, line.credit, digits=2) }}">
```

Comment:
@GavyMG 

Why not only
`
'%.2f' % move.debit`

Reply:
👍 changed.